book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative './rental'
+
+class Book
+  attr_accessor :title, :author, :rental
+
```

Comment:
### Optional suggestions

- [ ] I would recommend here to be `rentals` instead of a `rental`.